tests/plugins/chart/basic_chart.test.ts

```diff
@@ -1527,24 +1534,37 @@ describe("Chart design configuration", () => {
     expect(chart.chartJsConfig.data!.labels).toEqual(["3/1/2022", "2022/03/02"]);
   });
 
-  describe("Format of Y values at Runtime", () => {
-    function getTooltipItem(value: number, chartType: string): TooltipItem<ChartType> {
-      return {
-        label: "",
-        // yLabel is an number for pie charts for whatever reason
-        // @ts-ignore chart.js type is wrong
-        parsed:
-          chartType === "pie"
-            ? value
-            : {
-                x: 0,
-                y: value,
-              },
-        datasetIndex: 0,
-        index: 0,
-      };
-    }
+  function getTooltipLabel(
+    chart: ChartJSRuntime,
+    dataset: ChartDataset<
+      keyof ChartTypeRegistry,
+      (number | [number, number] | Point | BubbleDataPoint | null)[]
+    >,
```

Comment:
This argument seems useless, you can deduce it from chart and datasetIndex

chart.chartJsConfig.data!.datasets[datasetIndex]